File: iOS开发范例实战宝典(基础篇)/按钮/03-带下划线的按钮/03-带下划线的按钮/ViewController.m
//
//  ViewController.m
//  03-带下划线的按钮
//
//  Created by M-coppco on 16/5/28.
//  Copyright © 2016年 M-coppco. All rights reserved.
//

#import "ViewController.h"
#import "HJUnderLineButton.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    HJUnderLineButton *button = [[HJUnderLineButton alloc] init];

    [button setTitleColor:[UIColor blackColor] forState:(UIControlStateNormal)];
    [button setTitle:@"www.baidu.com" forState:(UIControlStateNormal)];
    button.frame = CGRectMake(10, 100, self.view.frame.size.width - 20, 30);
    [self.view addSubview:button];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift练习/从零开始学习Swift-关东生/第一部分  Swift语法篇/第三章 Swift语法基础/3.1__语法/3.1__语法/ViewController.swift
//
//  ViewController.swift
//  3.1__语法
//
//  Created by M-coppco on 16/6/23.
//  Copyright © 2016年 M-coppco. All rights reserved.
//

import UIKit

//声明了一个类
class Person {
    var name : String
    var age : Int
    init(name : String, age : Int) {
        self.name = name
        self.age = age
    }
}

class ViewController: UIViewController {

    

    override func viewDidLoad() {
        super.viewDidLoad()
       let person = Person(name: "王五", age: 20)
        person.name = "李四";person.age = 90
        //下面这句要修改,需要设置为var
//        person = Person(name: "张珊", age: 19)
        /**
        var str = "hello"
        print(str)
        str = "dsfdsd"
        */
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

